templates/server-statefulset.yaml

```diff
@@ -106,6 +106,18 @@ spec:
         - name: consul
           image: "{{ default .Values.global.image .Values.server.image }}"
           env:
+            - name: ADVERTISE_IP
+              valueFrom:
+                fieldRef:
```

Comment:
nit: I'd use `if true, else` instead of `if not true, else` just for less complexity. Unless this is following an existing pattern?

Reply:
I was copying from the client-daemonset, but I think I'll just switch both to `if true, else` because you're right, it'll be easier to read!